load_testing/ial2_sign_in.locustfile.py

```diff
@@ -0,0 +1,181 @@
+
+      """
+      Here's the flow:
+      1.  Login flow
+      2.  GET  /verify
+      3.  POST /verify/doc_auth/welcome
+      4.  POST /verify/doc_auth/upload?type=desktop
+      5.  POST /verify/doc_auth/front_image
+      6.  POST /verify/doc_auth/back_image
+      7.  POST /verify/doc_auth/ssn
+      8.  POST /verify/doc_auth/verify
+      9.  POST /verify/doc_auth/doc_success
+      10. POST /verify/doc_auth/phone
+      11. POST /verify/otp_delivery_method
+      12. POST /verify/phone_confirmation
+      13. POST /verify/review
+      14. POST /verify/confirmations
+      15. GET  /verify (redirects -> /verify/activated)
+      """
+
+
+      #  Sign in
+      do_sign_in(self)
+
+      #  Get /account page
+      dom, auth_token = get_request(self, "/account", "/account")
+
+      # Request IAL2 Verification
+      dom, auth_token = get_request(self, "/verify", "/verify/doc_auth")
+      
+      # Post consent to Welcome
+      dom, auth_token = post_request(self, 
+        "/verify/doc_auth/welcome",
+        { '_method': 'put',
+          'ial2_consent_given': 'true',
+          'authenticity_token': auth_token,
+        },
+        "/verify/doc_auth/upload")
+
+      # Choose Desktop flow
+      dom, auth_token = post_request(self, 
+        "/verify/doc_auth/upload?type=desktop",
+        { '_method': 'put',
+          'authenticity_token': auth_token,
+        },
+        "/verify/doc_auth/front_image")
+
+      # Post the Front Image of the license
+      front_path = sys.path[0] + "/load_testing/mont-front.jpeg"
+      with open(front_path, 'rb') as image, self.client.post(
+        "/verify/doc_auth/front_image",
+        headers= desktop_agent_headers(),
+        data={
+          '_method': 'put',
+          'authenticity_token': auth_token,
+          }, 
+        files={'doc_auth[image]': image},
+        catch_response=True) as resp:
+          verify_resp_url("/verify/doc_auth/back_image", resp)
+          dom = resp_to_dom(resp)
+          auth_token = authenticity_token(dom)
+      
+      # Post the Back Image of the license
+      back_path = sys.path[0] + "/load_testing/mont-back.jpeg"
+      with open(back_path, 'rb') as image, self.client.post(
+        "/verify/doc_auth/back_image",
+        headers= desktop_agent_headers(),
+        data={
+          '_method': 'put',
+          'authenticity_token': auth_token,
+          }, 
+        files={'doc_auth[image]': image},
+        catch_response=True) as resp:
+          verify_resp_url("/verify/doc_auth/ssn", resp)
+          dom = resp_to_dom(resp)
+          auth_token = authenticity_token(dom)
+
+      # SSN - use faker to get unique SSNs
+      fake = Faker()
+      ssn = fake.ssn()
+      print("*** Using ssn: " + ssn)
+      dom, auth_token = post_request(self, 
+        "/verify/doc_auth/ssn",
+        { '_method': 'put',
+          'authenticity_token': auth_token,
+          'doc_auth[ssn]': ssn,
+        },
+        "/verify/doc_auth/verify")
+
```

Comment:
The script finishes posting a SSN, and is redirected to the next page `verify/doc_auth/verify`, where we're just supposed to click "next" -- which posts back again to the same url.  Posting back always redirects me to `/account`, but I expect to be on `/verify/doc_auth/doc_success` 🤔 what am I missing?